src/AutorestSwift/View Models/ExceptionResponseViewModel.swift

```diff
@@ -32,6 +32,7 @@ struct ExceptionResponseViewModel {
     let objectType: String?
     /// Identifies the correct snippet to use when rendering the view model
     let description: String?
+    let isDefault: Bool
```

Comment:
This isn't very self-documenting. Is what default? Looks like it should be something like `isDefaultStatusCode`.  Is this a swagger feature where if someone doesn't set the status code for some operation it defaults to 200?

Reply:
In Swagger file, it just say 'default' with a definition schema '#/definitions/Error'

 ```
  "responses": {                                                                                                               
          "200": {                                                                                                                   
            "description": "The PNG file",                                                                                           
            "schema": {                                                                                                              
              "type": "file"                                                                                                         
            }                                                                                                                        
          },                                                                                                                         
          "default": {                                                                                                               
            "description": "Unexpected error",                                                                                       
            "schema": {                                                                                                              
              "$ref": "#/definitions/Error"                                                                                          
            }                                                                                                                        
          }                                                                                                                          
        }     
```

Reply:
Yes. good point. isDefaultStatusCode is a better name.